src/patterns/fundamentals/grid/react/info/stack-object.hbs

```diff
@@ -0,0 +1,94 @@
+<p class="sprk-u-mbm">See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+
+  <thead>
+    <tr>
+      <th>Component</th>
+      <th>Prop</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">SprkStack</td>
+      <td>children</td>
+      <td>String</td>
```

Comment:
Should the type be "SprkStackItem" for children? SprkTabs children prop type is listed as SprkTabsPanel